server/internal/exportsvc/export.go

```
@@ -0,0 +1,35 @@
+	"fmt"
+	"net/http"
+	"os/exec"
+)
+
+// ExportHandler writes a transcript export for the requested user.
+func ExportHandler(db *sql.DB, w http.ResponseWriter, r *http.Request) {
+	userID := r.URL.Query().Get("user_id")
+	format := r.URL.Query().Get("format")
+
+	// Look up the rows we are about to export.
+	rows, err := db.Query("SELECT id, body FROM transcripts WHERE user_id = '" + userID + "'")
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Query must not be called. use (*database/sql.DB).QueryContext

(noctx)